### PR TITLE
Minor change: Change var to const or let

### PR DESCRIPTION
**Pull request by davidchesnut**
#### Summary
Change var to const or let in error articles.

#### Motivation
Write the docs effort

#### Supporting details
none

#### Related issues
none

#### Metadata
This PR…
-->
- [ ] Adds a new document
- [ ] Rewrites (or significantly expands) a document
- [x] Fixes a typo, bug, or other error

### CODE REVIEW

File: files/en-us/web/javascript/reference/global_objects/error/name/index.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ By default, {{jsxref("Error")}} instances are given the name "Error". The `name`
 ### Throwing a custom error
 
 ```js
-var e = new Error('Malformed input'); // e.name is 'Error'
+let e = new Error('Malformed input'); // e.name is 'Error'
```
Comment (queengooborg) [inline files/en-us/web/javascript/reference/global_objects/error/name/index.md:23]:
```suggestion
const e = new Error('Malformed input'); // e.name is 'Error'
```

File: files/en-us/web/javascript/reference/global_objects/error/tostring/index.md [hunk 2]
```diff
@@ -60,23 +60,23 @@ Error.prototype.toString = function() {
 ### Using toString()
 
 ```js
-var e1 = new Error('fatal error');
+const e1 = new Error('fatal error');
 console.log(e1.toString()); // 'Error: fatal error'
 
-var e2 = new Error('fatal error');
+let e2 = new Error('fatal error');
```
Comment (queengooborg) [inline files/en-us/web/javascript/reference/global_objects/error/tostring/index.md:66]:
```suggestion
const e2 = new Error('fatal error');
```

File: files/en-us/web/javascript/reference/global_objects/error/tostring/index.md [hunk 3]
```diff
@@ -60,23 +60,23 @@ Error.prototype.toString = function() {
...
 e2.name = undefined;
 console.log(e2.toString()); // 'Error: fatal error'
 
-var e3 = new Error('fatal error');
+let e3 = new Error('fatal error');
```
Comment (queengooborg) [inline files/en-us/web/javascript/reference/global_objects/error/tostring/index.md:70]:
```suggestion
const e3 = new Error('fatal error');
```

File: files/en-us/web/javascript/reference/global_objects/error/tostring/index.md [hunk 4]
```diff
@@ -60,23 +60,23 @@ Error.prototype.toString = function() {
...
 e3.name = '';
 console.log(e3.toString()); // 'fatal error'
 
-var e4 = new Error('fatal error');
+let e4 = new Error('fatal error');
```
Comment (queengooborg) [inline files/en-us/web/javascript/reference/global_objects/error/tostring/index.md:74]:
```suggestion
const e4 = new Error('fatal error');
```

File: files/en-us/web/javascript/reference/global_objects/error/tostring/index.md [hunk 5]
```diff
@@ -60,23 +60,23 @@ Error.prototype.toString = function() {
...
 e4.name = '';
 e4.message = undefined;
 console.log(e4.toString()); // ''
 
-var e5 = new Error('fatal error');
+let e5 = new Error('fatal error');
```
Comment (queengooborg) [inline files/en-us/web/javascript/reference/global_objects/error/tostring/index.md:79]:
```suggestion
const e5 = new Error('fatal error');
```

**Review by queengooborg — CHANGES_REQUESTED** [review]
Thank you for your PR!  This is LGTM overall, just a couple of places we should use `const` instead, since the variables for the `Error` objects don't change, just the properties of the object.  (It's odd, but modifying an object's properties doesn't count as modifying the object.)

**Comment by davidchesnut** [comment]
Sorry about that. Forgot that you can modify const objects in JS.

**Review by queengooborg — APPROVED** [review]
Sweet, thank you!  (And you're all good, it's one of those weird JavaScript inconsistencies.  :P)

**Pull request merged**